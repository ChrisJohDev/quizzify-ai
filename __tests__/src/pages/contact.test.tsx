import {render, screen }from '@testing-library/react';
import Contact from '@/pages/contact';
import '@testing-library/jest-dom'

describe('Contact - snapshot test', () => {
  it('renders correctly', () => {
    const { container } = render(<Contact />);
    expect(container).toMatchSnapshot();
  });
});