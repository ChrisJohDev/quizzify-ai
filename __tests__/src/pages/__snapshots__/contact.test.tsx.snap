// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact - snapshot test renders correctly 1`] = `
<div>
  <div
    class="container flex flex-col mx-auto contactPage"
  >
    <h1
      class="text-center"
    >
      Contact Us
    </h1>
    <div
      class="col-md-6 offset-md-3 flex flex-col info"
    >
      <div
        class="mx-auto tableWrapper"
      >
        <table>
          <tbody>
            <tr>
              <td>
                Telephone:
              </td>
              <td>
                +46 - (0)31 123 45 67
              </td>
            </tr>
            <tr>
              <td>
                Email:
              </td>
              <td>
                support@quizzify-ai.com
              </td>
            </tr>
          </tbody>
        </table>
      </div>
      <form
        class="mx-auto"
      >
        <label
          for="issue"
        >
          What is your issue?
        </label>
        <select
          id="issue"
          name="issue"
        >
          <option
            value="bug"
          >
            Bug
          </option>
          <option
            value="feature"
          >
            Feature Request
          </option>
          <option
            value="other"
          >
            Other
          </option>
        </select>
        <label
          for="input-element"
        >
          Label Text:
        </label>
        <input
          id="input-element"
          name="input-name"
          type="text"
        />
        <label
          for="input-element1"
        >
          Label Text:
        </label>
        <textarea
          id="input-element1"
          name="input-name1"
        />
         
        <br />
        <input
          type="submit"
          value="Submit"
        />
      </form>
    </div>
  </div>
</div>
`;
